python/bolt-mcp/tests/conftest.py

```
@@ -0,0 +1,66 @@
+@pytest.fixture(autouse=True, scope="session")
+def _allow_db_access(request):
+    """Lift pytest-django's DB guard for the whole session, if the plugin is present.
+
+    These tests manage their own migrated file database (see above) and read it from
+    request-handler worker threads via ``sync_to_async`` — a connection pytest-django's
+    transaction-based ``django_db`` fixture would not cover. We just unblock raw access
+    rather than adopt its test-DB/transaction model.
+    """
+    try:
+        blocker = request.getfixturevalue("django_db_blocker")
+    except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)